Life-Pill/pharmacy-pos-frontend
# Update UI Enhancements and Bug Fixes

## Pull Request: Update UI Enhancements and Bug Fixes

### Changes Made
This pull request includes several enhancements and bug fixes:

- Updated item view table to improve readability and user interface.
- Added navbar profile picture for enhanced user personalization.
- Introduced a new person image to enrich user experience.
- Resolved various bugs and issues reported during testing.
- Fixed an error in the category component to ensure proper functionality.

### Manual Testing Done
Manual testing was conducted to verify the following:
- Item view table displays correctly with updated styling and layout.
- Navbar includes the newly added profile picture and behaves responsively.
- The new person image is integrated seamlessly across relevant UI components.
- Verified bug fixes to ensure issues are resolved as expected.
- Confirmed category component now operates without errors.

### Reviewer
- **Name:** Tharindu Jayawardhana
- **Position:** Frontend Developer at LifePill
